dart/lib/sentry_io.dart

```diff
@@ -1,2 +1,3 @@
+// ignore: invalid_export_of_internal_element
```

Comment:
Curious why we need this here since we have not changed anything in this file

Reply:
This is because we are exporting an internal class from the dart SDK that only the flutter SDK should use, let me check again.

Reply:
<img width="838" alt="Bildschirmfoto 2022-11-07 um 13 49 56" src="https://user-images.githubusercontent.com/3984453/200314915-1ec93e2b-beda-4bbd-9f1e-e191e097a246.png">

Reply:
See https://github.com/getsentry/sentry-dart/pull/1088/commits/3bc46a700288b76f09d7781f09b4569c7358d4f9